Automattic/wp-calypso
# Stats: Deprecate feature flag "stats/new-main-chart" within chart-related components

#### Proposed Changes

* Refactor component `StatModuleChartTabs` with the new structure.
* Refactor component `StoreStatsOrdersChart` with new styles.
* Refactor component `WordAdsChartTabs` with the new structure.
* Refactor component `Chart` with modernized styles.
* Remove the empty value inside the `Store` chart tabs.

| Before | After |
| --- | --- |
| <img width="697" alt="截圖 2022-11-30 下午12 28 28" src="https://user-images.githubusercontent.com/6869813/204707741-84b365cf-2691-4587-add6-c35dbd2bbaed.png"> | <img width="730" alt="截圖 2022-11-30 下午12 28 14" src="https://user-images.githubusercontent.com/6869813/204707767-3988e142-5071-4245-a8bd-69f6871e4767.png"> |

#### Testing Instructions



> Note: The instructions to show the Store and Ads pages can be found at https://github.com/Automattic/wp-calypso/pull/69661 and https://github.com/Automattic/wp-calypso/pull/69743.

* Spin up this change with the Calypso Live link.
* Navigate to the Stats `Traffic`, `Insights`, `Store`, and `Ads` pages.
* Ensure the styles are the same as the previous ones.

#### Pre-merge Checklist





Related to #70209
